ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/RemoveUnexpectedArgumentsFix.fs

```diff
@@ -17,7 +17,7 @@ type RemoveUnexpectedArgumentsFix(warning: NotAFunctionError) =
     let prefixApp = warning.PrefixApp
     
     override x.Text =
-        if prefixApp.FunctionExpression == expr then RemoveUnexpectedArgument else RemoveUnexpectedArguments
+        if prefixApp.FunctionExpression == expr.IgnoreParentParens() then RemoveUnexpectedArgument else RemoveUnexpectedArguments

```

Comment:
Would it be better to get the parent node if needed before creating the highlighting and store it instead?